docs/source/quickstarts/gke.md

```diff
@@ -56,6 +56,11 @@ node-pools create 'scyllaclusters' \
 --node-taints='scylla-operator.scylladb.com/dedicated=scyllaclusters:NoSchedule'
 :::
 
+## Kubernetes prerequisites
+
+Beginning with GKE version `1.32.1-gke.1002000`, the Ubuntu image used by GKE clusters no longer provides the `xfsprogs` package by default.
+This package is required to format the local NVMe disks used by ScyllaDB (see [xfsprogs section](../installation/kubernetes-prerequisites.md#xfsprogs) of the Kubernetes prerequisites page for more details).
```

Comment:
ditto

Reply:
also shouldn't the `## Kubernetes prerequisites` section also contain the kubelet part above?